React/Views/RCTView.m

```diff
@@ -897,6 +897,11 @@ - (void)displayLayer:(CALayer *)layer
   }
 
   RCTUpdateShadowPathForView(self);
+  
+#if TARGET_OS_OSX // [TODO(macOS ISS#2323203)
+  // clipsToBounds is stubbed out on macOS because it's not part of NSView
+  self.layer.masksToBounds = self.clipsToBounds;
+#endif // [TODO(macOS ISS#2323203)
```

Comment:
nit: `// [TODO(macOS ISS#2323203)`

Reply:
I'm not sure what the nit is - can you add a code "suggestion"?

Reply:
Sorry, I meant please change the comment to `// ]TODO(macOS ISS#2323203` (left bracket instead of right)